container_registry/containeranalysis/snippets/README.md

```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```

Comment:
Please remove `container_registry` directory and move the files under `containeranalysis`.